<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="LogInPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="login_edit_email"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="login_pb_fogot_pw">
         <property name="text">
          <string>Forget your password?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="login_pb_login">
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="login_edit_password"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="login_lable_pw">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="login_label_email">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="login_label_usericon">
         <property name="text">
          <string>UserIcon</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="login_pb_create_account">
         <property name="text">
          <string>Create an account</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="login_pb_demo_page">
         <property name="text">
          <string>Sign in as a Demo User</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SignUpPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLineEdit" name="signup_edit_firstName">
         <property name="placeholderText">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="signup_edit_lastName">
         <property name="placeholderText">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLineEdit" name="signup_edit_email">
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="signup_edit_phone">
         <property name="placeholderText">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLineEdit" name="signup_edit_pw">
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLineEdit" name="signup_edit_pw_confirm">
         <property name="placeholderText">
          <string>*Password</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="signup_pb_create_account">
         <property name="text">
          <string>Create account</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="singup_label_signupIcon">
         <property name="text">
          <string>Sign Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DemoPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="demo_label_demoicon">
         <property name="text">
          <string>Sign Up As Demo User</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="demo_pb_admin">
         <property name="text">
          <string>ADMIN</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="demo_pb_manager">
         <property name="text">
          <string>MANGER</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="demo_pb_dev">
         <property name="text">
          <string>DEVELOPER</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="demo_pb_tester">
         <property name="text">
          <string>TESTER</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="demo_pb_signIn">
         <property name="text">
          <string>Have an account? Sign In</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PasswordPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="pw_label_forgot_pw">
         <property name="text">
          <string>If you forgot password, Please enter Phone number that you signed up, and remake password!</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pw_edit_phone">
         <property name="placeholderText">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pw_edit_new_pw">
         <property name="placeholderText">
          <string>New Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pw_edit_new_pw_confirm">
         <property name="placeholderText">
          <string>Password*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pw_pb_change_pw">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
